src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,541 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/TempBuffer.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Christoffel.hpp"
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/CovariantDerivOfExtrinsicCurvature.hpp"
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ProjectionOperators.hpp"
+#include "PointwiseFunctions/GeneralRelativity/WeylPropagating.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/ErrorHandling/Assert.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+/// \brief Detailed implementation of Bjorhus-type boundary corrections
+namespace Bjorhus {
+/// @{
+/*!
+ * \brief Computes the expression needed to set boundary conditions on the time
+ * derivative of the characteristic field \f$v^{0}_{iab}\f$
+ *
+ * \details In the Bjorhus scheme, the time derivatives of evolved variables are
+ * characteristic projected. Constraint-preserving correction terms
+ * \f$T^{\mathrm C}_{ab}\f$, Sommerfeld condition terms for gauge degrees of
+ * freedom \f$T^{\mathrm G}_{ab}\f$, and terms constraining physical degrees of
+ * freedom \f$T^{\mathrm P}_{ab}\f$ are added here to the resulting
+ * characteristic (time-derivative) field:
+ *
+ * \f{align}
+ * \Delta \partial_t v^{-}_{ab} = T^{\mathrm C}_{ab} + T^{\mathrm G}_{ab}
+ *                              + T^{\mathrm P}_{ab}.
+ * \f}
+ *
+ * These terms are given by the following expressions from
+ * \cite Lindblom2005qh :
+ *
+ * \f{eqnarray}
+ * T^{\mathrm C}_{ab} &=& \frac{1}{4}
+ *      \left(k^c k^d l_a l_b - k^c l_b P^d_a - k^c l_a P^d_b - k^d l_b P^c_a
+ *            - k^d l_a P^c_b + 2 P^{cd} P_{ab}\right) \partial_t v^{-}_{ab}\\
+ *     &&+ \frac{1}{2} \lambda_{-}c^{\hat{0}-}_c \left(\frac{1}{2} l_a l_b k^c +
```

Comment:
This is indeed Eq. 64 of gr-qc/0512093. The reason it looked different is because this function was assuming that the user-input null vectors / one forms are `t^a (t_a) \pm n^a (n_a)` without the normalization by `1/sqrt(2)`. That is the convention spec uses in its BC code, but I realize it isnt what we want in spectre (e.g. in [`src/PointwiseFunctions/GeneralRelativity/InterfaceNullNormals.hpp`](https://spectre-code.org/group__GeneralRelativityGroup.html#ga88dc523e0343b4f3f01769d2ce5d8421). I have fixed both the documentation and the code to expect the normalizing factor of `1/sqrt(2)` in input null vectors and one forms.

One easy way of verifying that this change is correct is to look at the changes made to the unit test that compares against spec's output. The only  code change needed in that test was to divide the null vectors and one forms by `sqrt(2)`, and the test completes successfully, as it should.